app/main.py

```diff
@@ -1,3 +1,47 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: ("Distance", float, int)) -> "Distance":
```

Comment:
Change your type annotation to use an updated syntax
```suggestion
    def __add__(self, other: Distance | float | int) -> Distance:
```
In case you face errors, use `from __future__ import annotations` import.

Reply:
Thank you, i did it